.travis.yml

```diff
@@ -1,14 +1,11 @@
 language: go
 go:
   - 1.5.1
-branches:
-  only:
-    - master
 services:
   - docker
 sudo: required
 script:
-  - make bootstrap test
+  - make test docker-build
```

Comment:
`make bootstrap` just prints "nothing to do" and `make build` doesn't actually exist. I can add them here if that's more consistent though.